front/lib/api/tracker.ts

```diff
@@ -1,19 +1,74 @@
 import type { TrackerGenerationToProcess } from "@dust-tt/types";
 
 import { sendEmailWithTemplate } from "@app/lib/api/email";
+import { Authenticator } from "@app/lib/auth";
+import { TrackerConfigurationResource } from "@app/lib/resources/tracker_resource";
+import type { Logger } from "@app/logger/logger";
 
 const TRACKER_FROM_EMAIL = "dev@dust.tt";
 const TRACKER_FROM_NAME = "Bob Tracker"; // 😬
 
-export async function sendTrackerEmail({
+/**
+ * Tracker notification consumer logic.
+ * Processes a tracker notification: sends emails to recipients and consumes the tracker.
+ */
+export const processTrackerNotification = async ({
```

Comment:
This code was just moved out of the activity so I can play with the email easier with a command, not involving temporal, but no change on this logic!